python_modules/libraries/dagster-databricks/dagster_databricks/databricks_pyspark_step_launcher.py

```diff
@@ -342,6 +355,7 @@ def step_events_iterator(
         start_poll_time = time.time()
         done = False
         step_context.log.info("Waiting for Databricks run %s to complete..." % databricks_run_id)
+        yield from self.log_run_page_url(step_context, databricks_run_id)
```

Comment:
it's ab it strange for log_run_page_url to be a generator -- I think it'd be fine to just have it be called `self.get_databricks_run_dagster_event` and have it return a DagsterEvent (then just yield that event here)

Reply:
Yeah, that is a good point! Updated per recommendations 👍